apps/explorer/src/pages/transaction-result/Signatures.tsx

```diff
@@ -62,16 +70,25 @@ export function Signatures({ transaction }: Props) {
 	const deserializedTransactionSignatures = transactionSignatures
 		.map((signature) => toParsedSignaturePubkeyPair(signature))
 		.flat();
-
-	const userSignature = getSignatureFromAddress(deserializedTransactionSignatures, sender!);
-
+	const userSignatures = getSignaturesExcludingAddress(
+		deserializedTransactionSignatures,
+		gasData!.owner,
+	);
```

Comment:
lmk if theres a better way to do this - since with multisig this can be an array, just excluding the sponsored one is sufficient